mappings/net/minecraft/client/render/entity/model/SkullEntityModel.mapping

```diff
@@ -1,5 +1,11 @@
 CLASS dgf net/minecraft/client/render/entity/model/SkullEntityModel
-	METHOD a setRotationAngles (FFFFFF)V
+	FIELD a skull Ldhd;
+	METHOD <init> (IIII)V
+		ARG 1 textureU
+		ARG 2 textureV
+		ARG 3 textureWidth
+		ARG 4 textureHeight
+	METHOD a render (FFFFFF)V
```

Comment:
`renderWithAngles`? im not sure `render` is fully correct.

`setRotationAngles` is wrong though.

Reply:
I picked `render` for consistency with other model classes. Did think of maybe `orientAndRender` but ultimately decided against it.

I'm open to other options.